generator.js

```diff
@@ -114,7 +131,20 @@ module.exports = (mode) => {
   return data;
 };
 
-function array_of(size, start, end) {
+const generateDevBuild = () => {
```

Comment:
This generator build will be common for both prod and dev right?

Perhaps `generator` would be a better name? Thoughts?

Reply:
Ah, fixed :)